server/src/map/map.helper.js

```diff
@@ -0,0 +1,116 @@
+const map = require('./map.model');
+
+/**
+ * Get all active maps
+ * @return {maps} - return all maps
+ * @return {err} - return error
+ */
+let getAllMaps = () => {
+  return map
+    .find({})
+    .sort({label: 1})
+    .exec()
+    .then((maps) => Promise.resolve(maps))
+    .catch((err) => Promise.reject(err));
+};
+
+/**
+ * Get map by Id
+ * @param {ObjectId} id - map id
+ * @return {map} - return map
+ * @return {err} - return error
+ */
+let getMapById = (id) => {
+  return map
+    .findById(id)
+    .exec()
+    .then((map) => Promise.resolve(map))
+    .catch((err) => Promise.reject(err));
+};
+
+/**
+ * Get map by label
+ * @param {String} label - map label
+ * @return {maps} - return map
+ * @return {err} - return error
+ */
+let getMapByLabel = (label) => {
+  return map
+    .findOne({
+      label: new RegExp('^' + label + '$', "i")
+    })
+    .exec()
+    .then((map) => Promise.resolve(map))
+    .catch((err) => Promise.reject(err));
+};
+
+/**
+ * Add map
+ * @param {object} req - Request json object
+ * @return {map} - return map
+ * @return {err} - return error
+ */
+let addMap = (req) => {
+  let oMap = new map();
+  oMap.label = req.label;
```

Comment:
if I'm right it can be completed directly: // Map it's first-upper-case for constructors is better way
```
return new Map({owner, label, isShared}).save()
```